app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -206,7 +206,7 @@ const DiffViewer = createReactClass({
     return (
       <button
         onClick={this.showPreviousArticle}
-        className="button dark small"
+        className="button shift-previous dark small"
```

Comment:
Instead of introducing a new utility class here, we can re-use one of the many ways already in the CSS for getting the spacing to appear. The best one I can find is to add the `pull-right` class to this button, and to add the `margin` class to the first button.